modules/runtime/nuxeo-runtime-test/src/test/java/org/nuxeo/runtime/RuntimeInitializationPreprocessorTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+
+package org.nuxeo.runtime;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.HotDeployer;
+import org.nuxeo.runtime.test.runner.RuntimeFeature;
+
+@RunWith(FeaturesRunner.class)
+@Features({ RuntimeInitializationMockFeature.class, RuntimeFeature.class })
+public class RuntimeInitializationPreprocessorTest {
+
+    @Inject
+    protected HotDeployer hotDeployer;
+
+    @Test
+    public void testDeploymentPreprocessorErrors() throws Exception {
+        // messages have been cleared up by now, see actual test in RuntimeInitializationMockFeature
```

Comment:
What about putting the feature as a static class here? It will improve readability.

Reply:
I reverted that change as maven test fail on FeaturesLoader usage of the static inner class:

```
java.lang.IllegalAccessException: class org.nuxeo.runtime.test.runner.FeaturesLoader$Holder cannot access a member of class org.nuxeo.runtime.RuntimeInitializationPreprocessorTest$ErrorFeature with modifiers "public"
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:361)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:591)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.nuxeo.runtime.test.runner.FeaturesLoader$Holder.<init>(FeaturesLoader.java:54)
	at org.nuxeo.runtime.test.runner.FeaturesLoader.loadFeature(FeaturesLoader.java:124)
	at org.nuxeo.runtime.test.runner.FeaturesLoader.loadFeatures(FeaturesLoader.java:100)
	at org.nuxeo.runtime.test.runner.FeaturesRunner.<init>(FeaturesRunner.java:106)
```